recipes/ghc/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+ls
+if [ `uname` == Darwin ]
+then
+    export DYLD_LIBRARY_PATH="${PREFIX}/lib"
+    export STACK_ROOT="${SRC_DIR}/s"
+    ./configure --prefix=$PREFIX
+    make
+    make install
+else
+   ./configure --prefix=$PREFIX
+   make
+   make install
+fi
```

Comment:
Is there some small set of `make check` or similar that we can run?

Reply:
added make fasttest